### ISSUE TITLE
Rofi in window mode doesnt switch to selected application

### ISSUE THREAD

**Issue by saygo-png** [saygo-png/nixos#3]

**Comment by saygo-png** [comment]
ye idk it works now

**Comment by saygo-png** [comment]
it doesnt work on awesomewm, but works in hyprland. Its broken with default awesomewm config, so it must not be an issue with my awesomewm config...

**Comment by saygo-png** [comment]
```lua
client.connect_signal("property::urgent", function(c)
    c.minimized = false
    c:jump_to()
end)
```
Hack that fixes this, but afaik this should work properly out of the box... The buggy behaviour is still there, using wmctrl or xdotool still doesnt allow you to change workspaces. This should have been fixed in theory https://github.com/awesomeWM/awesome/issues/3047 so it might be a regression that should be reported to AwesomeWM

**Comment by saygo-png** [comment]
Fixed fully in c33f8b8c47133c7e810cf437077d6540e6a2d34c
